onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -549,6 +543,47 @@
     <string name="navigation_drawer_open">Open navigation drawer</string>
     <string name="navigation_drawer_close">Close navigation drawer</string>
 
+    <!-- Issue reporting implementation-->
+    <string name="region_dialog_Message2"> Are you in %1$s</string>
+    <string name="report_dialog_out_of_region_message"> Please select a bus stop.</string>
+    <string name="report_dialog_trip"> Please select a trip.</string>
+
+    <!-- report issue screen-->
+    <string name="ri_desc">Issue Description</string>
+    <string name="ri_button_camera">Camera</string>
+    <string name="ri_button_image">ADD IMAGE</string>
+    <string name="ri_button_gallery">Gallery</string>
+    <string name="ri_address_not_found">Address not found. Please check your address.</string>
+    <string name="ri_location_problem_info">Error getting location, please tap on map</string>
+    <string name="ri_location_problem">Error getting location</string>
+    <string name="ri_service_description_problem">Service description could not be found.</string>
+    <string name="ri_service_default">Choose an issue</string>
+    <string name="ri_service_stop">Stop Problem</string>
+    <string name="ri_service_trip">Trip Problem</string>
+    <string name="ri_user_name">First Name</string>
+    <string name="ri_user_lastname">Last Name</string>
+    <string name="ri_user_email">Email</string>
+    <string name="ri_user_phone">Phone Number</string>
+    <string name="ri_no_trip">No arrival times, please check back later.</string>
+    <string name="ri_anonymous_checkbox">Keep it anonymous</string>
+    <string name="ri_submitting_message">Submitting your report</string>
+    <string name="ri_unsuccessful_submit">An error occurred during submission, please try again later.</string>
+
+    <!-- report issue types screen-->
+    <string name="rt_infrastructure_problem">Report an Infrastructure Issue</string>
+    <string name="rt_infrastructure_problem_header">Infrastructure Issue</string>
+    <string name="rt_customer_service">Contact Customer Service</string>
+    <string name="rt_stop_problem">Report a Bus Stop Problem</string>
+    <string name="rt_arrival_problem">Report Arrival Time Problem</string>
+    <string name="rt_app_feedback">Send App Feedback</string>
+    <string name="rt_ideas">Ideas For New Features</string>
+    <string name="rt_message_infrastructure_problem">Missing bus stop, broken bench, etc.</string>
+    <string name="rt_message_customer_service">Driver compliments and complaints, lost &amp; found, etc.</string>
+    <string name="rt_message_stop_problem">Wrong bus stop, etc.</string>
+    <string name="rt_message_arrival_problem">Late bus, early bus, etc.</string>
+    <string name="rt_message_app_feedback">App crash, bugs.</string>
```

Comment:
Let's change this to `App crash, bugs, etc.` to match the other lists (i.e., add the `, etc.`).